aten/src/ATen/native/cuda/BinaryRemainderKernel.cu

```diff
@@ -20,6 +26,17 @@ void remainder_kernel_cuda(TensorIterator& iter) {
         return r;
       });
     });
+  }
+  else if (isComplexType(iter.dtype())) {
+    AT_DISPATCH_COMPLEX_TYPES(iter.dtype(), "remainder_cuda", [&]() {
+      gpu_kernel_with_scalars(iter,
+        []GPU_LAMBDA(scalar_t a, scalar_t b) -> scalar_t {
+          auto q = std::trunc(a.real() / b.real());
+          auto r = std::fmod(a.real(), b.real()); 
+          auto arg = (a - q * b) / std::abs(a - q * b);
+          return arg * r;
+        });
```

Comment:
It is sufficient to just return `a - q * b`

Reply:
Thanks @nikitaved. So remainder code will look like this:
```
std::complex<scalar_t> remainder(std::complex<scalar_t> a, std::complex<scalar_t> b) {
    auto q = std::trunc(std::abs(a) / std::abs(b));
    return a - q * b;
}
```